e2ev2/tests/workspace-create.test.js

```diff
@@ -58,7 +58,9 @@ browserTest('create a workspace', async browser => {
   const confirmButton =
     await page.waitForSelector('[role="button"][aria-label="Confirm Delete"]')
   await expect(confirmButton.evaluate(n => n.style.cursor)).resolves.toBe('not-allowed')
-  await page.type('[role="dialog"] input[placeholder="type DELETE to confirm"]', 'delete')
+  const confirmationTextSelector = '[role="dialog"] input[placeholder="type DELETE to confirm"]'
+  await page.waitForSelector(confirmationTextSelector)
+  await page.type(confirmationTextSelector, 'delete')
```

Comment:
Using fewer assignments, if you like:
```suggestion
  await page.waitForSelector('[role="dialog"] input[placeholder="type DELETE to confirm"]')
    .then(eh => eh.type('delete'))
```
This takes advantage of the fact that `waitForSelector` returns an element handle and that has a `type` method already scoped to itself.